repository.js

```diff
@@ -0,0 +1,175 @@
+import { api } from '@rocket.chat/sdk'
+import { createClient } from 'redis';
+let client;
+export async function createRedisClient() {
```

Comment:
Separate Redis and rocketchat api, due to not nested functionallity,
maybe we will use the redis for other logic (like server caching)